src/components/AddItem.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import { db } from '../lib/firebase';
+
+const AddItem = () => {
+  const [groceryItem, setGroceryItem] = useState('');
+  const [daysToPurchase, setDaysToPurchase] = useState(null);
+
+  const onGroceryItemInputChange = (e) => {
+    setGroceryItem(e.target.value);
+  };
+
+  const onSubmitHandler = (e) => {
+    e.preventDefault();
+    db.collection('shoppingList').add({
+      itemName: groceryItem,
+      daysToPurchase: daysToPurchase,
+      lastPurchasedDate: null,
+    });
+
+    setGroceryItem('');
```

Comment:
Can we reset the radio button value on form submission as well?

Reply:
😄 I think they are with line 20: ```setGroceryItem('')```

Reply:
I have updated the code to uncheck all radio inputs whenever the form is submitted: https://github.com/the-collab-lab/tcl-16-smart-shopping-list/pull/19/commits/bd58c9bb49f15b714768da3a86af5f5e5cd0820d :smile: